### ISSUE TITLE
AMD S7150 GPU OSX Support

### ISSUE THREAD

**Issue by T-Herrmann-WI** [dortania/bugtracker#319]
Hi, I'm working for a long time to get a S7150 (Tonga XT GL Device ID 0x6929) Card working in MacOS inside Server System with Proxmox as Hypervisor. Until now without success. I was where confused that the spoofing just by Proxmox was working fine in Windows and Linux VM's but not in MacOS VM. If I used native supported AMD RX460 oder RX560 It worked flawless. I read at some source the SSDT way doesn't work well with Hypervisors like Proxmox only with bare metal Hackintosh. 

I used latest version of OpenCore, Lilu, and other KEXTs .

There are serval ways to change the device-id, but the GPU is not working. The AMD Kexts will be loaded but not fully adapted to the GPU.

Did anyone manged a spoof of a device-id with Proxmox and Opencore?

I uploaded some screenshots here:  https://forum.proxmox.com/threads/amd-radeon-rx6950xt-spoofing-for-macos-vm.125901/

Thank you

Best Tim

**Comment by khronokernel** [comment]
According to your screenshots on the proxmox forum, you're testing against macOS Ventura. Apple dropped support for GCN 3 and older GPUs in this OS. You'll need to use Monterey (macOS 12) or older for native graphics support.

Regarding Device ID support, these are natively supported Tonga cards in Monterey's AMD9000Controller.kext:
```py
# AMDTongaGraphicsAccelerator - AMD9000Controller
0x6920,  # R9 M395/ M395X
0x6921,  # R9 M295X / M390X
0x6930,  # Unknown
0x6938,  # R9 380X / R9 M295X
0x6939,  # R9 285/380
```
* [Reference](https://github.com/dortania/OpenCore-Legacy-Patcher/blob/0.6.6/data/pci_data.py#L758-L769)

I unfortunately can't guide you further as I no longer work on Hackintoshes, but hopefully this information can be of use.

**Comment by T-Herrmann-WI** [comment]
Yes in principle correct, but with opencore legacy patcher with root patch restore the AMD9000Controller kext.

**Comment by khronokernel** [comment]
I'm familiar, I am the project lead. But my stance still remains, you should first prioritize getting acceleration on a native OS, before adding additional complexities with 3rd party utilities. Helps you narrow the scope of potential issues

**Comment by T-Herrmann-WI** [comment]
Of course but I also was unsuccessful on Monterey in 2022 with SSDT way or Proxmox device-id change in VM way.
